recipes/at-spi2-core/new/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
```

Comment:
Is it correct, the `new` folder?

Reply:
I created a new recipe for 2.45.1, because starting from this release, it also includes `at-spi2-atk` and `atk`, so the `provides` attribute is different from the previous versions. It's true that "new" is not a good folder name. I'd take any suggestion !

Reply:
You could use the version as name then. But indeed they changed it in a minor version?!

Reply:
yes. https://github.com/GNOME/at-spi2-core/blob/master/NEWS#L3. I could rename the folder new to 2.45.X, but it'll certainly be used for 2.46 etc., so it'd be weird.
The other solution is to remove all versions before 2.45.1, and keep the recipe in new. It's what I started to do, but I had remorse and re-added the old recipe in https://github.com/conan-io/conan-center-index/pull/11527/commits/59c6f1db4237acf5a0994079c3b25bad7ef1958e

Reply:
okay, let's keep `new` then. This is a very exceptional case. I can remember few times when some project updated a minor like a major.